#ifndef CONFIGURATIONFILE_H
#define CONFIGURATIONFILE_H

#include <QDialog>
#include <QLineEdit>
namespace Ui {
class ConfigurationFile;
}

class ConfigurationFile : public QDialog
{
    Q_OBJECT

public:
    explicit ConfigurationFile(QWidget *parent = 0);
    ~ConfigurationFile();

private slots:
    void on_pushButton_clicked();

    void on_pushButton_6_clicked();

    void on_pushButton_7_clicked();

    void on_pushButton_2_clicked();

    void on_pushButton_3_clicked();

    void on_pushButton_4_clicked();

    void on_pushButton_5_clicked();

private:
    Ui::ConfigurationFile *ui;
    QString GetFileName(QLineEdit *edit);
};

#endif // CONFIGURATIONFILE_H
